File: CS50x/pset6/cash/cash.py
while(True):
    # Inputting and validating amount of cash owned
    try:
        dollars = float(input("Change owed: "))
        if dollars >= 0:
            break
    except:
        pass

# Dollars to cents
cents = int(dollars * 100)

coins = 0

# Counting coins
for i in [25, 10, 5, 1]:
    coins = coins + cents // i
    cents = cents % i

print(coins)

File: CS50x/pset1/cash/cash.c
#include <cs50.h>
#include <stdio.h>
#include <math.h>

int main()
{
    float dollars;
    int cents, count = 0;

    //Taking input in dollars
    dollars = get_float("Change owed: ");
    //Dollars to cents
    cents = round(dollars * 100);

    //Counting coins till cents becomes 0
    while (cents != 0)
    {
        if (cents >= 25)
        {
            cents = cents - 25;
        }
        else if (cents >= 10)
        {
            cents = cents - 10;
        }
        else if (cents >= 5)
        {
            cents = cents - 5;
        }
        else
        {
            cents = cents - 1;
        }

        //Incrementing counter of coins each time loop runs
        count++;
    }

    //Printing count of coins
    printf("%d", count);
}
